package.json

```diff
@@ -13,11 +14,13 @@
     "gatsby-image": "^2.2.29",
     "gatsby-plugin-google-analytics": "^2.2.2",
     "gatsby-plugin-google-fonts": "^1.0.1",
+    "gatsby-plugin-intl": "^0.3.3",
```

Comment:
Handles switching between languages / navigation to a page in the right language when you click a link / letting a page know what language was selected